packages/mdc-tab-bar/index.js

```diff
@@ -84,13 +86,8 @@ class MDCTabBar extends MDCComponent {
     tabScrollerFactory = (el) => new MDCTabScroller(el)) {
     this.tabFactory_ = tabFactory;
     this.tabScrollerFactory_ = tabScrollerFactory;
-
-    this.tabList_ = this.getTabElements_().map((el) => this.tabFactory_(el));
-
-    const tabScrollerElement = this.root_.querySelector(MDCTabBarFoundation.strings.TAB_SCROLLER_SELECTOR);
-    if (tabScrollerElement) {
-      this.tabScroller_ = this.tabScrollerFactory_(tabScrollerElement);
-    }
+    this.tabList_ = this.instantiateTabs_(this.tabFactory_);
```

Comment:
Can we just pass `tabFactory`? Does `this.tabFactory_` even need to exist? Ditto for `this.tabScrollerFactory_`.

Reply:
Doesn't seem to be necessary in the base case.